angular-bind-html-compile.js

```diff
@@ -25,7 +25,7 @@
         };
     }]);
 
-    if (module && module.exports) {
+    if ('undefined' !== typeof module) {
```

Comment:
Could you change this to `if (typeof module !== 'undefined' && module.exports)` please?